conf/battle/feature.conf

```diff
@@ -103,3 +103,7 @@ feature.attendance: on
 // Private Airship System (Note 1)
 // Requires: 2018-03-21aRagexeRE or later
 feature.privateairship: on
+
+// Stylist UI (Note 1)
+// Requires: 2015-11-04aRagexeRE or later
+feature.stylistui: on
```

Comment:
It look like https://github.com/idathena/trunk/commit/4bdd552e65a710d0347247aa7c6affe76fc1159a
I think you forgot. Thank you idathena.

Reply:
It's should be also thanks hercules?

Reply:
shall address to hercules instead for the original code, but like @aleos89 said, it's better to use YAML format

Reply:
xD